wiki/en/Running-a-Server.md

```diff
@@ -290,17 +259,17 @@ It is highly recommended to test your Server on a **Public Directory** first so
 
 ### Port forwarding
 
-If you are running a Server on your home network, people from outside will not be able to see things inside it. So if you want to operate a Jamulus Server, you need to open a hole in your router's firewall to let Jamulus Clients connect to it.
+If you are running a Server on your home network, people from outside will not be able to see things inside it. If you want to operate a Jamulus Server, you need to set up your router's firewall to let Jamulus Clients connect to the internal Server.
 
-**The default port for the current version of Jamulus is UDP (not TCP) port 22124.**
+**Note:** The default port for the current version of Jamulus is **UDP** (not TCP) port **22124**.
 
 The exact setup of port forwarding differs for every router. For help see [portforward.com](https://portforward.com).
 
-Once you have your router set up, you can get your external (WAN) IP address e.g. by [using Google](https://www.google.com/search?q=what+is+my+ip). Give this address to your friends so they can connect to your Server (_but see also the note on dynamic DNS below_). You yourself should connect using the local network (LAN) address of the machine the Server is running on. If you are running a Client on the same machine as your Server, that would be `localhost` or `127.0.0.1`. 
+Once you have your router set up, you can get your external (WAN) IP address e.g. by [using Google](https://www.google.com/search?q=what+is+my+ip). Give this address to your friends so they can connect to your Server (_but see also the note on dynamic DNS below_). You yourself should connect using the local network (LAN) address of the machine the Server is running on. If you are running a Client on the same machine as your Server, that would be `localhost` or `127.0.0.1`.
 
 ## Dynamic DNS and why you will probably need it
 
-Most domestic Internet connections will change their IP address after a period (hours, days or weeks). To make it easier for people to connect to you, you may therefore also want set up a dynamic DNS address. You can do this on [the machine](https://www.online-tech-tips.com/computer-tips/ddns-dynamic-dns-service/) that you are running the Jamulus Server on, or preferably your router [may support it](https://www.noip.com/support/knowledgebase/how-to-configure-ddns-in-router/).
+Most domestic Internet connections will change their IP address after a period (hours, days or weeks). To make it easier for people to connect to you, you might want set up a "dynamic DNS address". You can do this on [the machine](https://www.online-tech-tips.com/computer-tips/ddns-dynamic-dns-service/) that you are running the Jamulus Server on, or preferably your router [may support it](https://www.noip.com/support/knowledgebase/how-to-configure-ddns-in-router/).
```

Comment:
This para needs splitting.  It needs to say:
- What is "Dynamic DNS" (as that's what the heading has first)
- Why you will probably need it (as that's what the heading says the section explains)

Anything else (such as how to set it up) is extra - and should be separate (same section, different para).

Reply:
I'd be fine with just deleting this section on the grounds that it's not necessary to run Jamulus and is too long/involved.

Reply:
I would at least like to have a hint to dynamic dns (most people don't know about it).

Reply:
OK, just a passing mention would be enough since "dynamic dns" will be easy easy enough to google for.

Reply:
Ok. See my edit.